aries_vcx/src/xyz/proofs/prover/prover_internal.rs

```diff
@@ -178,15 +173,16 @@ pub async fn build_rev_states_json(pool_handle: PoolHandle, credentials_identifi
                     (None, None)
                 };
 
-                let (_, rev_reg_def_json) = get_rev_reg_def_json(pool_handle, rev_reg_id).await?;
+                let rev_reg_def_json = ledger.get_rev_reg_def_json(rev_reg_id).await?;
 
-                let (rev_reg_id, rev_reg_delta_json, timestamp) = get_rev_reg_delta_json(pool_handle, rev_reg_id, from, to).await?;
+                let (rev_reg_id, rev_reg_delta_json, timestamp) = ledger.get_rev_reg_delta_json(rev_reg_id, from, to).await?;
 
-                let rev_state_json = prover::libindy_prover_create_revocation_state(
+                let rev_state_json = anoncreds.create_revocation_state(
+                    tails_file,
                     &rev_reg_def_json,
                     &rev_reg_delta_json,
+                    100, // todo - timestamp as 100?
```

Comment:
should we be using `timestamp` variable from above?

Reply:
yes, let's do that

for sake of 3rd party validation, afj is doing that as well
https://github.com/hyperledger/aries-framework-javascript/blob/7f37a6265dcc8310b78c14e59f38f1de19fed8c3/packages/core/src/modules/indy/services/IndyRevocationService.ts#L104